packages/cubejs-backend-shared/src/package.ts

```diff
@@ -70,7 +70,7 @@ export function requireFromPackage<T = unknown | null>(pkg: string, options?: Re
 }
 
 export function isSslKey(content: string) {
-  return content.startsWith('-----BEGIN RSA PRIVATE KEY-----');
+  return content.startsWith('-----BEGIN RSA PRIVATE KEY-----') || content.startsWith('-----BEGIN EC PRIVATE KEY-----');
```

Comment:
Could you please add a simple test in `package.test.ts` file with an example of EC key? Thanks

Reply:
Done